### DISCUSSION TITLE
How to get specific property from databucket array of objects using 'oneOf'?

### DISCUSSION THREAD

**Discussion by jpeter06** [mockoon/mockoon#1011] [Help] [answered]
Hello all,
just trying to figure out if it is posible to obtain one property of a random element in an array in a datebucker.
I am doing  this: `{{ oneOf (dataRaw 'dataBucket' ) }}`. I am obtaining an object, but I want to obtain one property **name**

I am trying to get a ramdom **name**.
It is posible?
my dataBucket is:
`[
  {"name": "zone", "code": "z1"},
  {"name": "zone2", "code": "z2"},
  {"name": "zone3", "code": "z3"}
]`

**Comment by 255kb** [comment, marked as answer]
You would need to use `lookup` from Handlebars:

`{{ lookup (oneOf (dataRaw 'invoice' )) 'name' }}`

**Reply by jpeter06** [reply]
thanks  :-)

**Reply by fcFn** [reply]
Thank you! 

Can we ***please*** put this somewhere in the data bucket docs? I know this is Handlebars proper feature, not Mockoon's, but someone who's not all that familiar with Handlebars will struggle just like the OP and me!

**Reply by 255kb** [reply]
There is no place for this right now, but there is a plan to add some sort of "recipes" page where you would find many things like this.
